src/test/java/org/sonar/plugins/clojure/sensors/ancient/AncientSensorTest.java

```diff
@@ -32,6 +32,8 @@
     private CommandRunner commandRunner;
 
     private AncientSensor ancientSensor;
+    private static final String LEIN_CMD =
```

Comment:
I think the sensors should be able to access the `LEIN_CMD `variable. This way you don't need to create the same variable in all of the tests. For instance: you should be able to get the variable by using `ancientSensor.LEIN_CMD`. That applies to the other sensors as well.